app/src/history.js

```diff
@@ -0,0 +1,5 @@
+import { createBrowserHistory } from 'history'
```

Comment:
i can't decide if i like this soln or feel it to be a bit hacky :)

honestly, I'm not quite sure where `history` is imported from... it's not an installed npm package. seems like it's built into the browser/js runtime... but we may find it better to polyfill